README.md

```diff
@@ -17,8 +17,7 @@ To build the project you need the following pre-requisites:
 ## 🖥 Building
 
 Clone the project and do the following:
-- Open `IntegrationExample.xcodeproj`.
-- Copy over the `MimiSDK` xcframework files into the `IntegrationExample/Frameworks` folder.
+- Update MimiSDK package to latest version
```

Comment:
i'd just delete the 2nd point and leave the first one in place
```suggestion
- Open `IntegrationExample.xcodeproj`.
```

Reply:
also can we remove the `- MimiSDK v9 frameworks.` point from the the `Requirements` section above? since they'll now be fetched by SPM, we also don't need to explicitly mention this there.